src/lib/i18n.ts

```diff
@@ -0,0 +1,86 @@
+import i18next from 'i18next';
+import { createI18nStore } from 'svelte-i18next';
+
+i18next.init({
+	lng: 'en',
+	resources: {
+		en: {
+			translation: {
+				nav: {
+					sessions: 'Sessions',
+					knowledge: 'Knowledge',
+					settings: 'Settings',
+					motd: 'Motd',
+					theme: { dark: 'Dark', light: 'Light' }
+				},
+				sessions: {
+					pageTitle: 'Sessions',
+					new: 'New session',
+					empty: 'No sessions',
+					session: 'Session',
+					name: 'Name',
+					content: 'Content',
+					prompt: 'Prompt',
+					run: 'Run',
+					delete: 'Delete session',
+					noSessionChosen: 'Create a new session or choose an existing one from the list',
+					promptPlaceholder: 'Write literally anything',
+					cantConnectToOllama: "Can't connect to Ollama server",
+					stopResponse: 'Stop response',
+					systemPrompt: 'System prompt',
+					goToKnowledge: 'Go to knowledge',
+					writePromptToStart: 'Write a prompt to start a new session',
+					ollamaConnectionError: "Couldn't connect to Ollama. Is the [server running](/settings)?",
+					genericError: 'Sorry, something went wrong.\n```\n${{error}}\n```',
+					somethingWentWrong: 'Sorry, something went wrong.'
```

Comment:
Let's move all of the errors: `cantConnectToOllama`, `ollamaConnectionError`, etc... to `errors: { }`